arbitrator/prover/src/value.rs

```diff
@@ -77,6 +77,16 @@ impl From<ArbValueType> for ValType {
     }
 }
 
+impl From<&ParamType> for ArbValueType {
```

Comment:
Is there a reason why we don't just use ArbValueType instead of ParamType? Is it a module importing issue?

Reply:
ArbValueType is defined in machine and can be a bunch of things that are not wasm types (like func pointer).
Code sharing between builder and source (as far as I can see) can only be done using an external library they both used.. so going for arbutil and a new small type seemed to make most sense.